scripts/ci/gitlab/pipeline/build.yml

```diff
@@ -21,7 +21,7 @@ build-linux-stable:
     # Ensure we run the UI tests.
     RUN_UI_TESTS: 1
   script:
-    - time cargo build --profile testnet --features pyroscope --verbose --bin polkadot
+    - time cargo build --profile testnet --features pyroscope,fast-runtime --verbose --bin polkadot
```

Comment:
This is just for testing, right?

If I'm not mistaken the binaries built at this stage are later published on DockerHub and then on Versi.

Reply:
No, I need this for the CI test. Would that be a problem though? 
> are later published on DockerHub

only for testing purposes
> on Versi

As long as we generate the chainspec using the binary without this flag, we should be fine. But even with it, I don't see many usages for Rococo of that flag (or `prod_or_fast!`), only for parachain leases management.